### PR TITLE
Reword/consistency, do not use `wallhack` term anymore.

### PR DESCRIPTION
**Pull request by sailro**
Nothing to change regarding configuration nor code, we are already using `EFT.Trainer.Features.Players` in the `trainer.ini` file.

### CODE REVIEW

**Comment by sailro** [comment]
@clevebitr if you can have a look at `FeaturePlayersDescription`, I'm not sure how to change the wording. For `FeaturePlayersName` I used `球员`

File: Properties/Strings.zh-cn.resx
Comment (clevebitr) [inline Properties/Strings.zh-cn.resx:1]: “球员”Should be translated as"显示玩家"

Comment (clevebitr) [reply Properties/Strings.zh-cn.resx:1]: "球员" It refers to someone who plays ball games

Comment (sailro) [reply Properties/Strings.zh-cn.resx:1]: Ok thanks :)

**Review by solareon — APPROVED** [review]

**Comment by solareon** [comment]
Is there no description change needed for zh-cn?

**Comment by clevebitr** [comment]
> zh-CN 不需要更改描述吗？

If you want to try to avoid using" wallhack" With words like this, you can Delete  "透视" , the description, which I think is the same as the new description in English

**Pull request merged**